src/raftstore/store/store.rs

```diff
@@ -430,6 +437,12 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                     }
                 }
 
+                if is_leader {
+                    // Notify pd quickly to let it update the region meta.
+                    self.heartbeat_pd(self.region_peers.get(&region_id).unwrap());
```

Comment:
no, we will meet compile error if `let p = self.region_peers.get(&region_id).unwrap()`, error:

```
cannot borrow `self.region_peers` as mutable because it is also borrowed as immutable
```